Add documentation for new extra attributes for fyta (#40410)
home-assistant/home-assistant.github.io: source/_integrations/fyta.markdown

- High
- Too High

> For the measurements sensors (light, moisture, salinity and temperature) the maximal/minimal acceptable and maximal/minimal good value is saved as an extra attribute of the sensor, so that these values can be used for automations or visualizations on the dashboard.

Reviewer: ```suggestion
For the measurement sensors (light, moisture, salinity, and temperature), the maximal/minimal acceptable and maximal/minimal good values are saved as an extra attribute of the sensor, so that these values can be used for automations or visualizations on the dashboard.
```

> For the measurement sensors (light, moisture, salinity, and temperature), the maximal/minimal acceptable and maximal/minimal good values are saved as an extra attribute of the sensor, so that these values can be used for automations or visualizations on the dashboard.

Reviewer: "maximal/minimal good": is that what they are called? Not maximal/minimal quality or something similar?
